### ISSUE TITLE
Error running from GUI - null parameter 'InputObject'

### ISSUE THREAD

**Issue by rejohnson76** [paulwetter/DocumentConfigMgrCB#95]
Getting this error running from the GUI.  Using Feb 18, 2022 version.  SCCM is 2107.

Write-HTMLTable : Cannot bind argument to parameter 'InputObject' because it
is null.
At C:\Temp\DocumentConfigMgrCB-master\DocumentCMCB.ps1:8283 char:38
+ ... InputObject ($ADRDeployments|Select-Object @{Name='Collection';expres ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Write-HTMLTable], ParameterBin
   dingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,W
   rite-HTMLTable

**Comment by paulwetter** [comment]
Odd. Do you somehow have an ADR with no deployments?

**Comment by rejohnson76** [comment]
Good catch!  I had an old disabled ADR with no deployment.  Deleted it and no more errors.  Might be an interesting item to report on instead of erroring?

Thanks for all your efforts,
Russell

**Comment by paulwetter** [comment]
I was thinking the same thing.  I'll update this weekend to make it leave a note of that in the doc.  Though, this does make me think of building a "Potential Action Items" list in the documentation or something too...  have to think about that.
